salt/utils/minions.py

```diff
@@ -201,7 +201,8 @@ def _check_glob_minions(self, expr, greedy):  # pylint: disable=unused-argument
         '''
         Return the minions found by looking via globs
         '''
-        return fnmatch.filter(self._pki_minions(), expr)
+        return {'minions': fnmatch.filter(self._pki_minions(), expr),
+                'missing': list()}
```

Comment:
```
In [4]: timeit.timeit('[]', number=100000)
Out[4]: 0.003401219997613225

In [5]: timeit.timeit('list()', number=100000)
Out[5]: 0.016737269001168897

In [6]: timeit.timeit('dict()', number=100000)
Out[6]: 0.014586324999982025

In [7]: timeit.timeit('{}', number=100000)
Out[7]: 0.004838772998482455
```
Basically using `list()` and `dict()` are slow and should be avoided for simple use cases

Reply:
:+1: I have replaced all the occurrences of `list()` with `[]`

Reply:
thanks!